### ISSUE TITLE
Consider archiving the repository

### ISSUE THREAD

**Issue by vrslev** [pedrokiefer/aiostomp#31]
Hey there!

At my work, we were successfully using aiostomp in production for a while. Since aiostomp is not active anymore (and I don't really like the API), I decided to write my own Python STOMP client.

I think it worked out fine, here it is: https://github.com/vrslev/stompman

Would you consider archiving this repository and putting a link to stompman in README.md?

**Comment by ooliver1** [comment]
I wouldn't imagine the stomp protocol changes often, so there isn't much need for this to be "active". You are free to have alternatives but not ask for them all to point to you.

**Comment by vrslev** [comment]
You are absolutely right, things should be made so they do not need to change. However this package doesn't have some things that one might want:

- proper typing support (Python not Rust, maintenance overhead is required to keep code up to date),
- support for multiple servers and connection recovery to first available server,
- [FastStream](https://faststream.airt.ai) integration,
- two-way heart-beating support
